packages/styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ai-apps/styles",
-  "version": "2.148.0-next.15",
+  "version": "2.148.0",
```

Comment:
same version comment as above. I think this applies to all the places the `version` change, but I'm not 100% on that.